src/javascript/class-scanner.ts

```diff
@@ -466,12 +466,12 @@ class PrototypeMemberFinder implements Visitor {
     if (babel.isAssignmentExpression(node)) {
       if (!type) {
         const detectedType =
-            closureType(node.right, sourceRange, this._document);
-        if (detectedType instanceof Warning) {
-          warnings.push(detectedType);
+            getClosureType(node.right, jsdocAnn, sourceRange, this._document);
+        if (!detectedType.successful) {
+          warnings.push(detectedType.error);
```

Comment:
I think we should drop the warning here. A method's type is Function and that's fine, yeah?